### PR TITLE
ref #309: document-manager-styling

### PR DESCRIPTION
**Pull request by ckleis**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no  
| Deprecations? | no 
| Fixed issues  | chameleon-system/chameleon-system#309
| License       | MIT

See status in Issue: 
https://github.com/chameleon-system/chameleon-system/issues/309

### CODE REVIEW

File: src/CoreBundle/Resources/views/snippets-cms/CMSDocumentManager/standard.html.twig [hunk 1]
```diff
@@ -7,54 +7,82 @@
...
-            {% endif %}
-
-            {% if recordID != '' %}
-                <button type="button" class="btn btn-sm btn-primary" onclick="chooseSelectedFiles();"><span class="btn-icon" style="background-image: url({{sPathTheme}}/images/icons/accept.png);">{{'chameleon_system_core.document_manager.action_select'|trans}}</span></button>
+<div{% if displayFull is not defined or displayFull == false %} class="p-4"{% endif %}>
+    <div class="row" style="height: 100%; margin-right: 0px;">
+        <div class="col-12 col-xl-3 mb-4" style="height: 100%;">
+            {% if displayFull is not defined or displayFull == false %}
+                <h3>
+                    {{'chameleon_system_core.cms_module_media_manager.folder'|trans}}
```
Comment (pixeljunkie) [inline src/CoreBundle/Resources/views/snippets-cms/CMSDocumentManager/standard.html.twig]: The translation key is from the media manager. Please add a copy for the document manager.

Comment (ckleis) [reply src/CoreBundle/Resources/views/snippets-cms/CMSDocumentManager/standard.html.twig]: ok, done

File: src/CoreBundle/Resources/translations/admin.en.xliff [hunk 2]
```diff
@@ -2202,6 +2202,10 @@
                 <source>chameleon_system_core.image_chooser.tree_node</source>
                 <target>Directory</target>
             </trans-unit>
+            <trans-unit id="chameleon_system_core.document_manager.folder">
```
Comment (umulmrum) [inline src/CoreBundle/Resources/translations/admin.en.xliff]: Please add a plural `s` at the end (also for the translated value).

Comment (ckleis) [reply src/CoreBundle/Resources/translations/admin.en.xliff]: done

**Comment by umulmrum** [comment]
Thanks @ckleis

**Pull request merged**